addons/spreadsheet/static/src/components/share_button/share_button.js

```diff
@@ -35,6 +35,8 @@ export class SpreadsheetShareButton extends Component {
         const data = await freezeOdooData(model);
         const url = await this.props.onSpreadsheetShared(data, model.exportXLSX());
         this.state.url = url;
-        browser.navigator.clipboard.writeText(url);
+        setTimeout(async () => {
```

Comment:
It probably doesn't matter because it never happens but `await onOpened()` does not wait the clipboard promise (we'd need to wrap it inside a `new Promise((resolve) => ...)`